src/libutil/git.cc

```diff
@@ -314,7 +318,11 @@ Mode dump(
         return Mode::Symlink;
     }
 
-    case SourceAccessor::tMisc:
+    case SourceAccessor::tChar:
+    case SourceAccessor::tBlock:
+    case SourceAccessor::tSocket:
+    case SourceAccessor::tFifo:
+    case SourceAccessor::tUnknown:
```

Comment:
```suggestion
```
We can just rely on the default case here (everything is unsupported except for the types handled above).

Reply:
I get an error saying not all the cases are met.

Reply:
I think this is a warning we enable in clang.